src/platforms/apple/troubleshooting.mdx

```diff
@@ -22,4 +22,9 @@ We recommend updating to at least [7.5.3](https://github.com/getsentry/sentry-co
 
 ## Duplicated Crashes Event With Screenshots
 
-Since the introduction of screenshots for crashes in version [7.20.0](https://github.com/getsentry/sentry-cocoa/releases/tag/7.20.0), projects with  'enableCaptureScreenshot' enabled received duplicated crash reports. Therefore, we recommend updating to at least [7.25.0](https://github.com/getsentry/sentry-cocoa/releases/tag/7.25.0) to fix this problem.
\ No newline at end of file
+Since the introduction of screenshots for crashes in version [7.20.0](https://github.com/getsentry/sentry-cocoa/releases/tag/7.20.0), projects with  'enableCaptureScreenshot' enabled received duplicated crash reports. Therefore, we recommend updating to at least [7.25.0](https://github.com/getsentry/sentry-cocoa/releases/tag/7.25.0) to fix this problem.
+
+## Errors silently don't deliver when using `use_frameworks! :linkage => :static`
```

Comment:
Hello @CannibalKush.
What kind of error are you talking about?
Is it crashes? Because this is not true, even using static I got my errors delivered.

Do you have a way to reproduce this?

Reply:
We had a small chat on [Discord](https://discordapp.com/channels/621778831602221064/750735628932612096/1088761395853668402).

When `Sentry.captureException(new Error())` the errors are not sent to Sentry in @CannibalKush s' app.

Reply:
The `dynamic` linking works correctly with `sentry-cocoa`, before we merge this we need to test it with RN Project.

Reply:
@krystofwoldrich - were you able to test this with RN?

Reply:
I'm sorry for the late response, yes, I've tried it. And static linking is working in our sample app. 

@CannibalKush Here is how I tested it.

```
cd sample-new-architecture
export USE_FRAMEWORKS=static
yarn pod-install-legacy-production # static linking doesn't work for the new architecture
# and then build in Xcode
```

If this is still an issue we will need a minimal reproducible example to debug this. But we can solve that in RN SDK repo. I will close this.